deploy/helm/values.yaml

```diff
@@ -282,20 +282,40 @@ tolerations:
   operator: Equal
   value: "true"
   effect: NoSchedule
+- key: node-role.kubernetes.io/master
+  operator: Exists
+  effect: NoSchedule
```

Comment:
should add another toleration, according to https://kubernetes.io/docs/reference/labels-annotations-taints/#node-role-kubernetes-io-control-plane-taint

```
- key: node-role.kubernetes.io/control-plane
  operator: Exists
  effect: NoSchedule
```